### PR TITLE
TST: `test_tooltip.py` convert to functional tests instead of class

### PR DESCRIPTION
**Pull request by attack68**
- [x] adds `pytest.fixture` instead of Class setup method.
- [x] removes `test_tooltip_reindex` since it is already being tested by `test_tooltip_render`
- [x] other tests no changes except adopting fixtures

### CODE REVIEW

File: pandas/tests/io/formats/style/test_tooltip.py [hunk 1]
```diff
@@ -7,99 +7,79 @@
...
-                DataFrame([["tooltip"]]),
-                css_class="other-class",
-                props="color:green;color:red;",
-            )
-            .render()
-        )
-        assert "#T__ .other-class {\n  color: green;\n  color: red;\n}" in s
+
+
+@pytest.fixture
```
Comment (jreback) [inline pandas/tests/io/formats/style/test_tooltip.py:19]: same, make this styler

put these in a `conftest.py` to share across all tests in this dir

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/tests/io/formats/style/test_tooltip.py (hunk 1 above)
Comment (attack68) [reply pandas/tests/io/formats/style/test_tooltip.py:19]: name changed.

currently the `df`and `styler` in different modules are different. Can do a followup to implement consistent and universal fixtures.

Comment (jreback) [reply pandas/tests/io/formats/style/test_tooltip.py:19]: oh i c, ok then

**Comment by jreback** [comment]
thanks @attack68

**Pull request merged**